source/install/deploy-bitnami.rst

```diff
@@ -0,0 +1,15 @@
+.. _deploy-bitnami:
+
+Deploy Mattermost on Bitnami
+=============================
+
+Through our partnership with Bitnami you can automatically deploy Mattermost Team Edition to a demonstration server for up to 1 hour, up to 10 times per platform, including: 
+
+- `Amazon Web Services <https://about.mattermost.com/deploy-aws>`_
+- `Oracle Cloud Platform <https://about.mattermost.com/deploy-oracle>`_ 
+- `Microsoft Azure <https://about.mattermost.com/deploy-azure>`_
+- `Google Cloud Platform <https://about.mattermost.com/deploy-googlecloud>`_
+
+Alternatively, you may also deploy to `Debian 8 VMware or Oracle VirtualBox virtual machines <https://about.mattermost.com/deploy-debianvirtualmachine>`_.
+
+If you have an existing account, or choose to create a new account, you can run your auto-deployed Mattermost instance as long as you would like.
```

Comment:
agree, here, these things change all the time.  It's best to assume they have an account and it's running in that account.